lib/modules/datasource/conda/index.ts

```diff
@@ -57,9 +71,23 @@ export class CondaDatasource extends Datasource {
       result.homepage = response.body.html_url;
       result.sourceUrl = response.body.dev_url;
 
+      response.body.files.forEach((file) => {
```

Comment:
are you sure files is always defined?

use for-of loop

Reply:
yes, conda package must have files, otherwise it won't ge index-ed and anaconda won't think it have such package. even all files is yanked it won't be removed from api response.